digital_asset_types/src/dao/scopes/asset.rs

```diff
@@ -300,6 +314,25 @@ pub async fn get_related_for_assets(
         }
     }
 
+    // When using getAssetByOwner or searchAssets with an owner_address to retrieve fungible assets, the 'owner' field was previously returned as empty.
+    // Instead of an empty value, we should now display the actual owner of the token account associated with the asset.
+
+    if show_owner_for_fungible {
+        let token_account_data: Vec<token_accounts::Model> = token_accounts::Entity::find()
+            .filter(token_accounts::Column::Mint.is_in(asset_ids))
+            .all(conn)
+            .await?;
```

Comment:
Mints don't have owners they have mint authority so owner doesn't make sense on fungible assets. I think we should just keep this as null.

Reply:
Well this idea was inspired from helius, as in place of owner they also show the owner of the token-account for that corresponding fungible-asset which is being searched using searchAsset or getAssetByOwner for a given owner_address.
But happy to close this one.